src/Components/App/App.tsx

```diff
@@ -6,25 +6,22 @@ import { Bookshelf, Book } from '../Bookshelf/Bookshelf'
 import './App.css';
 import { Switch, Route } from 'react-router-dom'
 
-
-type Props = {}
-
 type State = {
   list: {
     displayName: string,
     queryName: string
   }[] | [],
   error: string,
-  favorites: Book[] | null
+  favorites: Book[] | []
 }
 
-class App extends React.Component<Props, State> {
-  constructor(props: Props) {
+class App extends React.Component<{}, State> {
```

Comment:
i like this better to remove the unnecessary type above, and just add the empty object typing here.